src/components/search/index.tsx

```diff
@@ -18,21 +18,21 @@ const formatSearch = (searchTerm: string): string => {
 };
 
 export const SearchInput: React.FC = () => {
-  const [products, setProducts] = useState<Product[] | []>();
+  const [names, setNames] = useState<string[] | []>([]);
   const handleChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
     const searchTerm = e.target.value;
-
     try {
+      const productNames: string[] = [];
       const search = formatSearch(searchTerm);
       const searchData = await getSearchResults(search);
-      setProducts(searchData);
+      searchData.map((product) => productNames.push(product.name));
```

Comment:
could you combine lines 25 + 28 and do 
```
const productNames =  searchData.map((product) => product.name);
```
otherwise looks good to me :D

Reply:
Great tip, Thanks!